packages/govuk-frontend/src/govuk/assets/manifest.webmanifest

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Great to see this 😊

Probably want to confirm if this overrides **template.njk** so we don't lose:

1. `<meta name="theme-color">`
2. `<link rel="mask-icon">`

For compatibility with web servers, it's often named **manifest.json**

Worth a check?

Reply:
Currently checking 'em! I think the HTML tags are safe, as neither are being overridden by the manifest.

Noted the .json file thing in my comment on the issue. It seems like `.webmanifest` is the preferred option these days, but I figure we might have to ask around on that to see how severely it might impact folks. Given this would be new for v5 I don't think it'll be a massive barrier to ask folks to check their content-type configurations?

Reply:
As an optional thing we might be alright?

We're shipping our default ESM bundle as `govuk-frontend.bundle.js` not `*.mjs` for content-types though!

Reply:
Yeah, I don't think this is worth causing a hubbub about and JSON is the 'safer' of the two options, so will go with that unless we have compelling reasons to do otherwise.